packages/bsky/src/api/app/bsky/notification/updateSeen.ts

```diff
@@ -12,6 +12,21 @@ export default function (server: Server, ctx: AppContext) {
         actorDid: viewer,
         timestamp: Timestamp.fromDate(new Date(seenAt)),
       })
+      await ctx.courierClient.pushNotifications({
+        notifications: [
+          {
+            id: `${viewer}-${seenAt}`,
```

Comment:
Sorry for the late review here!  Would prefer to avoid accepting user input as the id and make it something more deterministic.  All the other notif ids are murmur hashes with parts separated by double-colons, may make sense to recreate that here.

```ts
import murmur from 'murmurhash'

const getNotifId = (viewer: string, seenAt: Date) => {
  const key = ['mark-read-generic', viewer, seenAt.getTime().toString()].join(
    '::',
  )
  return murmur.v3(key).toString(16)
}
```

Reply:
Ah, lovely. thanks for pointing in the right direction!